modules/network-api/src/main/resources/application.yml

```
@@ -8,34 +8,33 @@ spring:
     port: ${FRONTEND.NETWORK_API.PORT}
-
 main-api:
   base-url: ${MAIN_API_BASE_URL}
-  api-key: ${MAIN_API_API_KEY}
\ No newline at end of file
+  api-key: ${MAIN_API_API_KEY}
+
+#### 아래는 다른 모듈의 설정
+infra-jpa:
+  domain:
+    enabled: true
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)